app/src/main/res/values/strings.xml

```diff
@@ -106,6 +106,8 @@
     <string name="example_pass_123">pass@123</string>
     <string name="remove_dialog">REMOVE</string>
     <string name="password_remove">Password Removed</string>
+    <string name="file_date_text">File Date</string>
```

Comment:
Add these strings to files of other locales also. That might fix the build error

Reply:
Okay.